test/jest_integration/saml_auth.test.ts

```diff
@@ -302,6 +302,8 @@ describe('start OpenSearch Dashboards server', () => {
     await driver.executeScript('arguments[0].scrollIntoView(true);', radio);
     await driver.executeScript('arguments[0].click();', radio);
 
+    await new Promise((r) => setTimeout(r, 500));
```

Comment:
Rather than sleep can we wait for a specific element to be visible / interactable that will let us know that the line on 307 will succeed?

Reply:
@peternied This is now updated to wait for selection.